### PR TITLE
Maint hoodies & Stack files standardization

### PR DESCRIPTION
**Pull request by PestoVerde322**
## About The Pull Request

This PR adds hoodies in the game, in many colours and also, including a Durathread craftable variant, also standardize all the timers under the stack folder to use `SECONDS` instead of deciseconds.

## Why It's Good For The Game

Done on bacon's request as extra drip is required.

## Testing Photographs and Procedure
<details>
<summary>Screenshots&Videos</summary>

immagine

immagine

immagine

immagine

</details>

## Changelog
:cl:
add: Added hoodies, avviable via the beecoin shop, in-game vendors or crafting
add: Added Durathread hoodies, avviable only via crafting
code: Standardized all the timers under the stack folder to use `SECONDS` instead of deciseconds and some True-False variables
/:cl:

### CODE REVIEW

**Comment by Fronsis** [comment]
Another goated Pigeon W PR

**Comment by PowerfulBacon** [comment]
image
Is the hood over the backpack while on the head?

**Comment by PestoVerde322** [comment]
> Is the hood over the backpack while on the head?

apparently so, will check

**Review by PowerfulBacon — DISMISSED** [review]

**Comment by Tsar-Salat** [comment]
This shit's nice. 

Maybe using wirecutters on it makes it sleeveless? That'd be sick for engie's showing off their gains.

Or possibly one's with departmental colors?

**Comment by itsmeow** [comment]
could you move the pocket on the item sprite down one pixel? it looks like the fuckin among us helmet protrusion as it is

**Comment by Fronsis** [comment]
> could you move the pocket on the item sprite down one pixel? it looks like the fuckin among us helmet protrusion as it is

among us and it's consequences on society + brainrot

**Comment by PestoVerde322** [comment]
> could you move the pocket on the item sprite down one pixel? it looks like the fuckin among us helmet protrusion as it is

now that you mentioned it, i won't change it.

**Comment by Tsar-Salat** [comment]
im going to port these to tg and call them "among us" hoodies and see what happens

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 1]
```diff
@@ -394,8 +395,10 @@ GLOBAL_LIST_INIT(durathread_recipes, list ( \
 	new/datum/stack_recipe("durathread beret",						/obj/item/clothing/head/beret/durathread, 2, time = 40), \
 	new/datum/stack_recipe("durathread beanie",						/obj/item/clothing/head/beanie/durathread, 2, time = 40), \
 	new/datum/stack_recipe("durathread bandana",					/obj/item/clothing/mask/bandana/durathread, 1, time = 25), \
+	new/datum/stack_recipe("durathread hoodie",						/obj/item/clothing/suit/hooded/hoodie/durathread, 5, time = 50), \
```
Comment (Archanial) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: use time defines

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 2]
```diff
@@ -394,8 +395,10 @@ GLOBAL_LIST_INIT(durathread_recipes, list ( \
...
 	))
 
+
```
Comment (Archanial) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]:
```suggestion
```

**Review by Archanial — CHANGES_REQUESTED** [review]

File: code/game/objects/items/stacks/sheets/sheet_types.dm (hunk 1 above)
Comment (PestoVerde322) [reply code/game/objects/items/stacks/sheets/sheet_types.dm]: Fixed to all the files under stack.

**Review by PowerfulBacon — APPROVED** [review]
It arrived

**Review by Archanial — APPROVED** [review]

**Pull request merged**